### PR TITLE
Add doc with info about binary upload a custom route, for #500

### PR DESCRIPTION
**Pull request by jonathanbp**
Add a doc with custom route on mux for binary file upload.

### CODE REVIEW

**Comment by jonathanbp** [comment]
Yes! Sorry about this mess, @johanbrandhorst. It turns out I had the wrong email set in my gitconfig. It should be better now.

File: docs/docs/mapping/binary_file_uploads.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+	err := r.ParseForm()
+	if err != nil {
+		http.Error(w, fmt.Sprintf("failed to parse form: %s", err.Error()), http.StatusBadRequest)
+		return
+	}
+
+	f, header, err := rq.FormFile("attachment")
+	if err != nil {
+		http.Error(w, fmt.Sprintf("failed to get file 'attachment': %s", err.Error()), http.StatusBadRequest)
+	}
```
Comment (johanbrandhorst) [inline docs/docs/mapping/binary_file_uploads.md:45]: Missing a return here

**Review by johanbrandhorst — CHANGES_REQUESTED** [review]
Thanks for your patience with our bot 😅. One small thing before we can merge!

File: docs/docs/mapping/binary_file_uploads.md (hunk 1 above)
Comment (jonathanbp) [reply docs/docs/mapping/binary_file_uploads.md:46]: Should be added now. Had another round with the bot also.

**Review by johanbrandhorst — APPROVED** [review]

**Comment by johanbrandhorst** [comment]
Thanks for your contribution!

**Pull request merged**